scripts/make_release.sh

```diff
@@ -17,11 +17,11 @@ set -euo pipefail
 
 ################################################################################
 # Release-specific parameters (Change when you bump the version)
-OLD_GIT_TAG="v2020-09-30"
-OLD_CABAL_VERSION="2020.9.30"
+OLD_GIT_TAG="v2020-10-13"
+OLD_CABAL_VERSION="2020.10.13"
 
-GIT_TAG="v2020-10-13"
-CABAL_VERSION="2020.10.13"
+GIT_TAG="v2020-11-03"
+CABAL_VERSION="2020.11.03"
```

Comment:
Actually, think this should be `2020.11.3`, and that it is complaining because of this

Reply:
(The cabal version only)

Reply:
Yes, only recent versions of cabal do support numbers with leading zero. Although we arguably require an oddly old version of cabal our cabal files.